torch/_decomp/decompositions.py

```diff
@@ -431,6 +427,26 @@ def _nll_loss_backward(
     return grad_input * grad_output
 
 
+@register_decomposition(aten.addcmul_)
+def addcmul_(input: Tensor, tensor1, tensor2, value=None, out=None):
```

Comment:
use @out_wrapper() instead. Also we have a new style for these; open a PR for the new decomp to master, then merge it into the branch.

Reply:
sg

Reply:
This will not be necessary after https://github.com/pytorch/pytorch/pull/88117 is merged.

Reply:
sg :)